setup.py

```diff
@@ -34,12 +34,12 @@
     extras_require={
         'docs': [
             'sphinx != 1.3.1',
-            'nbsphinx',
-            'sphinx_rtd_theme==0.5.*',
-            'sphinx-multiversion >= 0.2.3',
-            'sphinx-gallery >= 0.7',
-            'spinxcontrib-svg2pdfconverter',
-            'presets'
+            'sphinx_rtd_theme == 0.5.*',
+            'nbsphinx == 0.8.*',
+            'spyder == 4.2.*',
```

Comment:
my spyder is 4.0.1, my numpydoc is 0.9.2,  so we can probably lower the version?